arlo-client/src/components/Audit/Progress/JurisdictionDetail.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { Classes, Dialog } from '@blueprintjs/core'
+import FormButton from '../../Atoms/Form/FormButton'
+import { IJurisdiction } from '../useJurisdictions'
+
+export interface IDialogExampleState {
+  autoFocus: boolean
+  canEscapeKeyClose: boolean
+  canOutsideClickClose: boolean
+  enforceFocus: boolean
+  isOpen: boolean
+  usePortal: boolean
+}
+
+const JurisdictionDetail = ({
+  isOpen,
+  handleClose,
+  jurisdiction,
+  electionId,
+}: {
+  isOpen: boolean
+  handleClose: () => void
+  jurisdiction: IJurisdiction
+  electionId: string
+}) => {
+  if (!jurisdiction) return null // component still loading data
+  return (
+    <Dialog
+      icon="info-sign"
+      onClose={handleClose}
+      title={`${jurisdiction.name} Audit Information`}
+      isOpen={isOpen}
+    >
+      <div className={Classes.DIALOG_BODY}>
+        <p>
+          <strong>Ballot Manifest Upload Status: </strong>
+          {jurisdiction.ballotManifest.processing
+            ? jurisdiction.ballotManifest.processing.status
+            : 'N/A'}
```

Comment:
I think this could be improved in a couple ways to be more useful to the AA:
- Prettify the statuses (maybe factor this bit out into a function and reuse it: https://github.com/votingworks/arlo/blob/jurisdiction-detail/arlo-client/src/components/Audit/Progress/index.tsx#L53-L63)
- In the error case, show the error message in `processing.error`